### DISCUSSION TITLE
v1.2.1

### DISCUSSION THREAD

**Discussion by OrangeJedi** [OrangeJedi/Aerial#151] [Announcements]
## This new release includes the following:
* H.265 Video Support (#17)
    * Aerial now supports Apple's H.265 videos in both 1080p (for lower file sizes) and 4K
* There are now two ways to have Aerial avoid launching when playing other videos: (#114)
    * Use `aerial.scr` to install using the system screensaver API (There is a possibility you will encounter high CPU usage and low performance and some app features, like launching with a global shortcut won't work.)
    * Or configure Aerial to run as an administrator
* Minor bug fixes (#149, #138)

#### Self Launching App
Download `Aerial Setup 1.2.1.exe` (or download the .zip and extract it) and run it. Aerial takes care of the rest. A configuration window will open where you can adjust the settings.
#### Windows Screensaver
Download `aerial.scr`, copy it to a folder where you won't delete it, right click, and click on install. Use the dialogue to configure settings.

#### Things to note
* Disable any other screensavers before installing
* Set your computer sleep settings to longer than the screensaver start time
* If you want your computer to download and cache the videos, you must set up the video cache manually

<hr /><em>This discussion was created from the release <a href='https://github.com/OrangeJedi/Aerial/releases/tag/v1.2.1'>v1.2.1</a>.</em>

**Comment by gitanO0** [comment]
After installing this, my PC completely shuts down instead of sleeping.  Is there a config item I can change to have it sleep?  All my OS power management settings remain unchanged and under no circumstances do those settings have the PC shut down.

**Reply by gitanO0** [reply]
Update:  If I disable all the built-in sleep functions within the screen saver config app, and set my own sleep config in the OS, things work properly.  Seems to be something within the app config that is causing the PC to shut down rather than sleep.

**Comment by Hajker** [comment]
The only thing that is missing is HDR videos. I hope they can get added one day, but the software is already great as is! Big thank you to who made it!

**Comment by mateousss** [comment]
Hi, after installing it on my Windows 10, nothing happens, Aerial does not open. The Aerial settings won't open neither (the settings button is greyed out after I click on it...). Do you know why? Screenshot of the issue is attached.
![Issue](https://github.com/user-attachments/assets/1783147f-b581-437b-81fa-3000f8d1d66e)

**Comment by iuseaminipc** [comment]
are you not gonna update this anymore?
